microservices/ga/datamgr/cheatsheet.md

```diff
@@ -60,18 +60,18 @@
 
 # Resources
 
-Can be accessed using their short name or full name.  Multiple resources can be requested depending on the kubernetes service/control being used.
+Can be accessed using their short name or full name.  Multiple resources can be requested depending on the Kubernetes service/control being used.
 
 ```bash
-kubectl get po,deploy,svc 
+kubectl get pod,deploy,svc 
```

Comment:
po is the short name for pod

Reply:
I was just going for readability, all of them can easily be changed back